lib/global_uid/migration_extension.rb

```diff
@@ -8,15 +8,12 @@ def create_table(name, options = {}, &blk)
       # rules for stripping out auto_increment -- enabled, not dry-run, and not a "PK-less" table
       remove_auto_increment = uid_enabled && !GlobalUid::Base.global_uid_options[:dry_run] && !(options[:id] == false)
 
-      if remove_auto_increment
-        old_id_option = options[:id]
-        options.merge!(:id => false)
-      end
+      options.merge!(:id => false) if remove_auto_increment
 
       super(name, options) { |t|
         if remove_auto_increment
           # need to honor specifically named tables
-          id_column_name = (old_id_option || :id)
+          id_column_name = options.fetch(:id_column_name, :id)
```

Comment:
Can we just remove this feature entirely? What’s the risk? Could we use the documented `primary_key` parameter to `#create_table` instead?